src/Php80/bootstrap.php

```diff
@@ -20,23 +20,23 @@
 }
 
 if (!function_exists('fdiv')) {
-    function fdiv(float $dividend, float $divisor): float { return p\Php80::fdiv($dividend, $divisor); }
```

Comment:
Could we conditionally define these functions so that on PHP 7 they have the types?

Reply:
yes, updated.